py/mpconfig.h

```diff
@@ -597,6 +597,10 @@
 #define MICROPY_READER_VFS (0)
 #endif
 
+#ifndef MICROPY_READER_VFS_DEFAULT_BUF_SIZE
+#define MICROPY_READER_VFS_DEFAULT_BUF_SIZE (48)
```

Comment:
This should be 39 or 55 -- `sizeof(mp_obj_t)+2*sizeof(uint16_t)+sizeof(uint8_t)+MICROPY_READER_VFS_DEFAULT_BUF_SIZE` needs to be a multiple of the block size.